package.json

```diff
@@ -32,6 +32,7 @@
     "bounding-client-rect": "1.0.5",
     "browser-filesaver": "1.1.0",
     "chalk": "1.0.0",
+    "check-node-version": "2.0.1",
```

Comment:
Not a typo, `2.0.1` was the latest version until 2 days ago :)

Bumped to the latest.